### PR TITLE
Remove hrl

### LINKED ISSUE #2: Simplify .travis.yml
Have this working:
```
language: erlang
otp_release:
  - 19.3
  - 20.3
  - 21.3
  - 22.3
  - 23.0
script: make travis
```

### LINKED ISSUE #3: Add badges to README
build |
erlang >=
as per eredis.

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
I started by wanting to remove the .hrl from this lib.

I did that.

Then I wanted to modernize this a bit.

I think I did that.

Also:
* closes #3,
* closes #2.

### CODE REVIEW

File: test/sbroker_SUITE.erl [hunk 1]
```diff
@@ -204,7 +204,7 @@ skip_down_match(_) ->
             Self = self(),
             {await, Ref, _} = sbroker:async_ask_r(Broker, Self, {Self, Ref}),
             sys:resume(Broker),
-            {drop, _} = sbroker:await(Ref, ?TIMEOUT),
+            _ = sbroker:await(Ref, ?TIMEOUT),
```
Comment (g-andrade) [inline test/sbroker_SUITE.erl]: ^ Why the change to the matched return value?

Comment (paulo-ferraz-oliveira) [reply test/sbroker_SUITE.erl]: Tests fail. Should we look closer into it? Or just assume it's working as is and move on?

Comment (g-andrade) [reply test/sbroker_SUITE.erl]: I think we should look into it; I believe they were working before (e.g. under OTP 19.)

Comment (g-andrade) [reply test/sbroker_SUITE.erl]: Alas, it appears I've at one point known about this issue:

https://github.com/fishcakez/sbroker/pull/5#issuecomment-500453110

Comment (paulo-ferraz-oliveira) [reply test/sbroker_SUITE.erl]: So, is this just a "test fail, but code doesn't" type of situation, or is there something we REALLY want to fix, here?

Comment (g-andrade) [reply test/sbroker_SUITE.erl]: You tell me... OTP behaviour has changed at a fundamental level; as a consequence, the way the test case was written means something else now.

But something is still being tested here. Whatever it is, either we ensure that it keeps working (which likely requires us to rewrite the test in order to accommodate for the new constraints at the VM level) or we say, right here and right now, that it's too much of a hassle and trust that whatever feature is being tested will not break - which is an acceptable compromise, as long as we're aware of what we're skipping.

Comment (paulo-ferraz-oliveira) [reply test/sbroker_SUITE.erl]: I'll try to find time to look at it later, then. I don't want to sweep it under the rug "just because".

Comment (paulo-ferraz-oliveira) [reply test/sbroker_SUITE.erl]: As discussed, we'll go with "skip this test case": dc47fe8

**Review by g-andrade — APPROVED** [review]
